tests/unit/test_cassettes.py

```diff
@@ -29,6 +29,19 @@ def test_cassette_load(tmpdir):
     assert len(a_cassette) == 1
 
 
+def test_cassette_load_nonexistent():
+    a_cassette = Cassette.load(path="something/nonexistent.yml")
+    assert len(a_cassette) == 0
+
+
+def test_cassette_load_invalid_unicode(tmpdir):
+    a_file = tmpdir.join("invalid_unicode.yml")
+    with open(a_file, "wb") as fd:
+        fd.write(b"\xda")
```

Comment:
Just a detail: I would prefer wording "invalid encoding" (or "malformed encoding") over "invalid Unicode" here, to be more precise.

Reply:
Good point, done!